cypress/support/step-definitions/tabs-steps.js

```diff
@@ -0,0 +1,19 @@
+import { tabs, tabById, tabContentById } from '../../locators/tabs';
+
+const TAB_CONTENT = 'Content for tab ';
+
+Then('Tabs component is align to {string}', (value) => {
+  tabs().should('have.css', 'text-align', value);
+});
+
+Then('Tabs component posistion is set to {string}', (value) => {
+  tabs().should('have.css', 'flex-direction', value);
```

Comment:
flex-direction -> should be a const

Reply:
we can use const for css assert if const is used more than one